deptrac.yaml

```
@@ -77,6 +77,14 @@ deptrac:
           collectors:
               - type: className
                 regex: ^Composer\\*
+      -   name: HttpClients
+          collectors:
+            - type: className
+              value: ^Symfony\\*
```

Could/should this be more specific? 💭 
```suggestion
              value: ^Symfony\\Component\\HttpClient\\*
```